blog/2024-01-23/index.md

```diff
@@ -0,0 +1,58 @@
+
+## What's Driving Growth
+
+We attribute the growth of Flathub to a few developments over the years, and especially the last several months:
+
+1. **Popular app availability**: Since we started Flathub, we've seen apps like Firefox, Google Chrome, Discord, VLC, Spotify, Telegram, Microsoft Edge, Steam, OBS Studio, Zoom, Thunderbird, and _many_ more make their way into the store. These are many of the apps people want and expect, especially if they're coming to a Linux desktop for the first time. Each time another popular app hits Flathub, it makes Flathub that much more compelling to users.
+
+2. **Verified apps**: We've heard that some folks have held off installing specific apps or even using Flathub altogether because they didn't want a third-party maintainer redistributing an app they rely on—verifying apps solves this by assuring users their favorite app is actually coming from its developer. With an ever-increasing number of apps choosing to get verified—including both big names like Firefox, Discord, and Telegram as well as many newer indie apps—the trust and adoption of Flathub increases.
+
+3. **Steam Deck**: It's not [just a meme](https://mastodon.blaede.family/@cassidy/111031129234702967); we suspect Flathub being included as the default app source for the Steam Deck's desktop mode has had a large positive effect on the usage of Flathub. Just look at some of the [most popular apps](https://flathub.org/apps/collection/popular): retro game emulators, game compatibility tools, gaming-oriented chat services, alternate game launchers—all the kinds of apps you'd expect Steam Deck users to want. And according to our category [statistics](https://flathub.org/statistics), "games" itself is the most downloaded category of app on Flathub. It turns out selling ["multiple millions" of devices](https://www.theverge.com/2023/11/9/23954205/valve-steam-deck-multiple-millions) has an impact on the ecosystem!
+
+4. **Linux distro adoption**: In addition to shipping on Steam Deck, Flathub now comes included out of the box on at least Clear Linux, Endless OS, KDE Neon, Linux Mint, Pop!_OS, and Zorin OS—and [as of Fedora 38](https://pagure.io/fesco/issue/2939), Flathub is available in its entirety when enabling third-party software sources. **Flathub is the preferred app store for Linux, and its grassroots adoption across the Linux desktop ecosystem proves that.**
+
+5. **New apps**: Last but definitely not least, we've entered a thrilling chapter of Flathub as an ecosystem. It's not just another source to get popular, big-name apps you'd expect to be able to get anywhere else; **Flathub has become the preferred app store for a growing number of indie, open source developers**. Developers are increasingly submitting their own new apps and directing their users to Flathub.
```

Comment:
Wondering, how big the role of builder directly bootstrapping a flatpak for you is.

Reply:
I could hypothesize at the end of this paragraph if you think that's useful?

```suggestion
5. **New apps**: Last but definitely not least, we've entered a thrilling chapter of Flathub as an ecosystem. It's not just another source to get popular, big-name apps you'd expect to be able to get anywhere else; **Flathub has become the preferred app store for a growing number of indie, open source developers**. Developers are increasingly submitting their own new apps and directing their users to Flathub. And tools like GNOME Builder make that easier than ever with [first-class Flatpak support](https://developer.gnome.org/documentation/introduction/flatpak.html) built in.
```